README.md

```diff
@@ -87,6 +119,18 @@ class AppView extends View
 }
 ```
 
+## BootstrapUI Layout
+
+BootstrapUI comes with it's own `layout.ctp` file and examples taken from the Twitter Bootstrap framework.
+
+When no layout for the view is defined the `BootstrapUI\View\UIViewTrait` will load it's own `layout.ctp` file. You can
+override this behavior in two ways.
+
+- Assign a layout to the view with `$this->layout('layout.ctp')`.
+- Disable auto loading of the layout in `BootstrapUI\View\UIViewTrait` with `$this->initializeUI(['layout'=>false]);`.
+
+### Loading the Twitter Bootstrap framework
```

Comment:
@ADmad ah okay. I wanted to somehow differentiate between the framework and this plugin. I'm not exactly sure how to do that because they are both named bootstrap.